src/main/kotlin/com/openlattice/data/DataGraphService.kt

```diff
@@ -379,6 +380,17 @@ open class DataGraphService(
 
 
     override fun getNeighborEntitySets(entitySetId: UUID): List<NeighborSets> {
-        return lm.getNeighborEntitySets(entitySetId)
+        return graphService.getNeighborEntitySets(entitySetId)
+    }
+
+    override fun getNeighborEntitySetIds(entitySetId: UUID): Set<UUID> {
+        return getNeighborEntitySets(entitySetId)
+                .asSequence()
+                .flatMap { sequenceOf(it.srcEntitySetId, it.edgeEntitySetId, it.dstEntitySetId) }
```

Comment:
Just making sure you intended to use asSequence here. Sequences cannot be inlined by the compiler and are for lazy evaluation. Given that NeighborSets being returned will most likely fit in memory it's more efficient to flat map with a listOf(...)

Reply:
I used `AnalysisService` as an example:

```
val neighborEntitySets = dgm.getNeighborEntitySets(entitySetId)
val entitySetIds = neighborEntitySets.asSequence()
        .flatMap { sequenceOf(it.srcEntitySetId, it.edgeEntitySetId, it.dstEntitySetId) }
        .toSet()
```

Reply:
Yeah, that's my bad. I shouldn't have used sequence there.